appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm

```diff
@@ -434,12 +443,37 @@
                          registeredComponentName eventName)
                        #f))))
 
+       ;; For the HandlesEventDispatching interface
+       (define (dispatchMultiEvent eventName :: java.lang.String
```

Comment:
It would be more consistent to call these generic events rather than
multievents.



== Hal Abelson
hal@mit.edu
Prof. of Comp. Sci. and Eng.
MIT Dept. of Elec. Eng. and Comp. Sci.
MIT Media Lab, Second Class Faculty


On Sat, Apr 11, 2015 at 12:58 PM, gitars <notifications@github.com> wrote:

> In
> appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm
> <https://github.com/mit-cml/appinventor-sources/pull/347#discussion_r28196542>
> :
>
> > @@ -434,12 +443,37 @@
> >                           registeredComponentName eventName)
> >                         #f))))
> >
> > +       ;; For the HandlesEventDispatching interface
> > +       (define (dispatchMultiEvent eventName :: java.lang.String
>
> Yea, sorry for that confusion, I've been using "multi-generic event" and
> "multi-event" pretty interchangeably.
>
> Also, yea, I created a new Yail "define-multi-event" for those blocks
> (still in the same function that creates the Yail for normal events and
> uses "define-event"). And then, I pretty much tried to mimic everything
> that happens for define-event, for define-multi-event, which doesn't happen
> an instance associated with it, just a the component type.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mit-cml/appinventor-sources/pull/347/files#r28196542>.
>